tests/read-varint-unsigned-long.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+Test that reading 64-bit VarInt works fine.
+--DESCRIPTION--
+VarInt::readingUnsignedLong(ByteBuffer) processes up to 9 consecutive 7-bit bytes (a 63-bit integer) until the last processed byte has an MSB (most significant bit).
```

Comment:
This is actually 10 bytes, but the mode of failure should kick in with anything more than 5 anyway